core/utils/poly-look/src/react/tabs/swipableTabs.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from "react";
+import { Swiper, SwiperSlide } from "swiper/react";
+import Tab from "./tab.jsx";
+
+import "./tabs.css";
+
+const SwipableTabs = ({ children }) => {
```

Comment:
Looking at the code of `Tabs` and this one, I think it could be cleaner to just have a boolean `swipe` attribute. The easiest approach seems to be to use `Swiper` in both cases, but just disable the swipe gesture in one. But can be refactored later, if that makes sense, for now I think it's fine.

Reply:
Also typo, I think: `SwipeableTabs`?

Reply:
yeah need to check, when you put one slide only it lets you move it around, which I didn't like. But I'll see if I can disable that

Reply:
yeah need to check, when you put one slide only it lets you move it around, which I didn't like. But I'll see if I can disable that